platform/darwin/src/MGLPointCollection.mm

```diff
@@ -68,7 +68,7 @@ - (MGLCoordinateBounds)overlayBounds
 
 - (BOOL)intersectsOverlayBounds:(MGLCoordinateBounds)overlayBounds
 {
-    return MGLLatLngBoundsFromCoordinateBounds(_bounds).intersects(MGLLatLngBoundsFromCoordinateBounds(overlayBounds));
+    return MGLCoordinateBoundsIntersectsCoordinateBounds(_bounds, overlayBounds);
```

Comment:
I’ve updated MGLMultiPolygon and MGLMultiPolyline to use the simplified expression too.